src/rlookup.c

```diff
@@ -599,10 +599,25 @@ static RSValue *replyElemToValue(RedisModuleCallReply *rep, RLookupCoerceType ot
   }
 }
 
+static bool isValueAvailable(const RLookupKey *kk, const RLookupRow *dst, RLookupLoadOptions *options)
+{
+  if (!options->forceLoad && kk && !(kk->flags & RLOOKUP_F_FORCE_LOAD)) {
```

Comment:
I don't think we need to validate `kk` here

1. I don't think it's ever NULL
2. If it was NULL, this condition yields `false`, and the next condition (else if) will dereference it and we will crash